.github/dependabot.yml

```diff
@@ -36,8 +36,3 @@ updates:
     directory: /
     schedule:
       interval: daily
```

Comment:
Just removed /terraform/paas/ but why not change to /terraform/aks/

Reply:
done